README.md

```diff
@@ -1,4 +1,21 @@
-# Python Boilerplate For GitHub Tasks
+# Music School
 
 **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
-before starting.
+before start.
+
+Welcome to our Music School.
```

Comment:
Welcome to our Music School! Many musicians apply to us every day, and we need you to create a model for it named **Musician**.